CONTRIBUTING.md

```diff
@@ -1,7 +1,24 @@
 # Contributing
 
-You can take part in making Shopsys Framework better.
+You can take part in making Shopsys Framework Demoshop better.
```

Comment:
Do we really want developers to contribute to demoshop? In which way? The only case we would want to accept some is when somebody would create feature for us. Will this even happen?

Reply:
I do not knwo if this will happend but I do not mind if this happend. And I am open to PR. It can for example contains fix of some issue.